db/migrate/20200213205538_add_initial_facets.rb

```diff
@@ -0,0 +1,61 @@
+class AddInitialFacets < Mongoid::Migration
+  def self.up
+    SearchFacet.find_or_create_by!(name: "species") do |facet|
+      facet.identifier = "species"
+      facet.is_ontology_based = true
+      facet.is_array_based = false
+      facet.big_query_id_column = 'species'
+      facet.big_query_name_column = 'species__ontology_label'
+      facet.convention_name = 'alexandria'
+      facet.convention_version = '1.0'
+      facet.ontology_urls = [{name: 'todo', url: 'https://to.do/ontology'}]
```

Comment:
We can pull the ontology URLs from the [JSON](https://github.com/broadinstitute/scp-ingest-pipeline/blob/master/tests/data/AMC_v1.1.3.json) schema for all of these.